framework/src/Volo.Abp.Http.Client.IdentityModel/Volo/Abp/Http/Client/IdentityModel/IdentityModelRemoteServiceHttpClientAuthenticator.cs

```diff
@@ -20,7 +20,7 @@ public virtual async Task Authenticate(RemoteServiceHttpClientAuthenticateContex
         {
             await IdentityModelAuthenticationService.TryAuthenticateAsync(
                 context.Client,
-                context.RemoteService.GetIdentityClient()
+                context.RemoteServiceName
```

Comment:
How about `context.RemoteService.GetIdentityClient() ?? context.RemoteServiceName`?

Reply:
@maliming Hi:
I though code inside  `context.RemoteService.GetIdentityClient()`  is wrong
```csharp
    public static class RemoteServiceConfigurationExtensions
    {
        public const string IdentityClientName = "IdentityClient";
        public const string UseCurrentAccessTokenName = "UseCurrentAccessToken";

        [CanBeNull]
        public static string GetIdentityClient([NotNull] this RemoteServiceConfiguration configuration)
        {
            Check.NotNullOrEmpty(configuration, nameof(configuration));

            return configuration.GetOrDefault(IdentityClientName);
        }
}
public class RemoteServiceConfiguration : Dictionary<string, string>
    {
        /// <summary>
        /// Base Url.
        /// </summary>
        public string BaseUrl
        {
            get => this.GetOrDefault(nameof(BaseUrl));
            set => this[nameof(BaseUrl)] = value;
        }

        /// <summary>
        /// Version.
        /// </summary>
        public string Version
        {
            get => this.GetOrDefault(nameof(Version));
            set => this[nameof(Version)] = value;
        }

        public RemoteServiceConfiguration()
        {
            
        }

        public RemoteServiceConfiguration(string baseUrl, string version = null)
        {
            this[nameof(BaseUrl)] = baseUrl;
            this[nameof(Version)] = version;
        }
    }
```
 `RemoteServiceConfiguration` seems did't has `IdentityClient` key,
So,I don't know why use the  `context.RemoteService.GetIdentityClient()` method to get `RemoteServiceName`,Maybe we should check with @hikalkan 
On the other hand,
should we define `appsetting.json` like below?(I did't test it)
```json
{
  "RemoteServices": {
    "Default": {
      "BaseUrl": "https://localhost:44301/"
    },
    "MyProjectName": {
      "BaseUrl": "https://localhost:44300/",
      "IdentityClients":"MyProjectName"
    }
  },
  "IdentityClients": {
    "Default": {
      "GrantType": "password",
      "ClientId": "MyProjectName_App",
      "ClientSecret": "1q2w3e*",
      "UserName": "admin",
      "UserPassword": "1q2w3E*",
      "Authority": "https://localhost:44301/",
      "Scope": "MyProjectName"
    },
    "MyProjectName": {
      "GrantType": "password",
      "ClientId": "MyProjectName_App",
      "ClientSecret": "1q2w3e*",
      "UserName": "admin",
      "UserPassword": "1q2w3E*",
      "Authority": "https://localhost:44301/",
      "Scope": "MyProjectName"
    }
  }
}
```